results_next.csv

```diff
@@ -1,155 +1,155 @@
 Benchmark cycles: 1,000 5,000 10,000 50,000 100,000 500,000 1,000,000 5,000,000
```

Comment:
**NOTE**: The numbers here are lower, which is due in a small part to the addition of the word `string` as a prefix (more characters to process === slower processing times), however it is also due do a difference in machines. The numbers before all changes were lower than existing as well.